### PR TITLE
Prevent RemovedInDjango19Warning on Django1.8

### PR DESCRIPTION
**Pull request by giginet**
On Django.1.8, `django.db.models.get_model` moved to `django.db.models.loading.get_model`. So I added version checking before loading `get_model`.

### CODE REVIEW

File: django_comments/views/comments.py [hunk 1]
```diff
@@ -16,6 +17,12 @@
 from django_comments.views.utils import next_redirect, confirmation_view
 
 
+if django.VERSION < (1, 8):
+    from django.db.models import get_model
+else:
+    from django.db.models.loading import get_model
```
Comment (collinanderson) [inline django_comments/views/comments.py]: isn't it now django.apps.apps.get_model?

https://docs.djangoproject.com/en/1.8/ref/applications/

One possibility: `from django.apps import apps` in 1.8, and in 1.7: `from django.db import models as apps`. Then `apps.get_model()` in the code

Comment (claudep) [inline django_comments/views/comments.py]: +1
See also https://github.com/claudep/django-reversion/commit/4e1e178482e4c7f4dc8c19e2097d26daef488cc1

**Comment by giginet** [comment]
@collinanderson @claudep Thank for reviewing. I fixed.

**Comment by claudep** [comment]
Thanks, but I'd prefer the `try: ... except ImportError: ...` way of doing things. Then we don't have to inspect versions.

**Comment by giginet** [comment]
@claudep Sorry for your checking many times.
How is this?

**Comment by claudep** [comment]
You don't need to be sorry :-) Yes, that's what I meant. Now you can remove the django import, and then merge your commits.

**Comment by giginet** [comment]
@claudep I've fixed. Please merge it.

**Comment by claudep** [comment]
Thanks, I merged the commits myself and pushed in 45347d0ae8

**Pull request closed without merging**